src/uu/install/src/install.rs

```diff
@@ -1042,3 +1043,13 @@ fn need_copy(from: &Path, to: &Path, b: &Behavior) -> UResult<bool> {
 
     Ok(false)
 }
+
+#[cfg(feature = "selinux")]
+fn set_selinux_context(path: &Path, behavior: &Behavior) -> UResult<()> {
+    if !behavior.preserve_context && behavior.context.is_some() {
+        // Use the provided context set by -Z/--context
+        set_selinux_security_context(path, behavior.context.as_ref())
+            .map_err(|e| InstallError::SelinuxContextFailed(e.to_string()))?
```

Comment:
clippy complains about a missing `;` (see https://github.com/uutils/coreutils/actions/runs/14981525054/job/42190579881?pr=7908#step:12:222)
```suggestion
            .map_err(|e| InstallError::SelinuxContextFailed(e.to_string()))?;
```